composer.json

```diff
@@ -25,22 +25,17 @@
     "issues": "https://github.com/postcode-nl/PostcodeNl_Api_Client/issues",
     "support": "https://github.com/postcode-nl/PostcodeNl_Api_Client"
   },
-  "repositories": [
-    {
-      "type": "vcs",
-      "url": "https://github.com/postcode-nl/PostcodeNl_Api_Client"
-    }
-  ],
   "autoload": {
-    "psr-0": {
-      "PostcodeNl\\Api": "src/"
-    },
-    "classmap": [
-      "src/PostcodeNl/Api/Client.php"
-    ]
+    "psr-4": {
+      "PostcodeNl\\Api\\": "src"
+    }
+  },
+  "replace": {
+    "postcode-nl/api-restclient": "*"
```

Comment:
While PostcodeNl_Api_Client does replace api-restclient, it is not a drop-in replacement which "replace" would imply.

Reply:
Replace in composer just means that if you also have the original package in composer that one should not be installed. But I can leave this out if you want.

Reply:
As far as I know, "replace" means you tell composer that this package contains the features of the original package and version, i.e. it has its classes and can be used instead of the original package. This is mainly used when you fork a package or have a bundle of multiple packages in one package. 

Which is not the case here, so it would be best not to have it.